packages/core/app-extensions/src/login/sagas.js

```diff
@@ -17,18 +14,3 @@ export function doRequest(url, options) {
 export function* doSessionRequest() {
   return yield call(doRequest, 'session', {method: 'POST'})
 }
-
-export function* sessionCheck() {
-  const {success, businessUnit, adminAllowed} = yield call(doSessionRequest)
-  const cachedPrincipal = cache.getLongTerm('session', 'principal')
-  if (cachedPrincipal && cachedPrincipal.currentBusinessUnit.id !== businessUnit) {
-    yield call(cache.clearAll)
-  }
-  yield put(actions.setAdminAllowed(adminAllowed))
-  yield put(actions.setLoggedIn(success))
```

Comment:
So before we set `adminAllowed` before `loggedIn`. This caused that `adminAllowed` always had a value when Admin is rendered.
Now we set `loggedIn` separately and only set `adminAllowed` when we already render `Admin` component. This causes the prop types of `Admin` to be unhappy because `adminAllowed` prop is required.

I still try to understand all these changes.. nevertheless either the proptypes are not correct anymore or we need to be more careful with setting the loggedIn and adminAllowed values

Reply:
`adminAllowed` is set to `undefined` in the file `packages/apps/admin/src/modules/session/sagas.js`. So I think should the proptype was wrong